instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/patches/server.rb

```diff
@@ -14,10 +14,10 @@ def request(op, *args)
             operation = Utils.opname(op, multi?)
             attributes = {
               'db.system' => 'memcached',
-              'db.statement' => Utils.format_command(operation, args),
               'net.peer.name' => hostname,
-              'net.peer.port' => port
+              'net.peer.port' => port,
             }
+            attributes['db.statement'] = db_statement if config[:db_statement] == :include
```

Comment:
I think if you do the following, you can remove the additional code below and then you're just left with adding a test case.  
```suggestion
            attributes['db.statement'] = Utils.format_command(operation, args) if config[:db_statement] == :include
```